server2/Arcadia.Assistant/Arcadia.Assistant.WorkHoursCredit/WorkHoursCredit.cs

```diff
@@ -80,6 +80,16 @@ public async Task<WorkHoursChange[]> GetCalendarEventsAsync(EmployeeId employeeI
             return events;
         }
 
+        public async Task<Dictionary<EmployeeId, WorkHoursChange[]>> GetCalendarEventsByEmployeeMapAsync(EmployeeId[] employeeIds, CancellationToken cancellationToken)
+        {
+            using var ctx = this.dbFactory();
+            var events = await this.QueryCalendarEvents(ctx.Value.ChangeRequests, x => employeeIds.Any(id => id.Value == x.EmployeeId))
+                .ToArrayAsync(cancellationToken);
+
+            return events.GroupBy(x=> x.EmployeeId)
```

Comment:
Not done. Code is still not formatted.

x=> x.EmployeeId

Should be a space before an arrow.

Reply:
Done again